packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -192,6 +193,26 @@ export default class Lexis extends Module {
       console.warn('Lookup request cannot be completed: LexisCS configuration is unavailable')
       return
     }
+    const featureOptions = this._settingsApi.getFeatureOptions()
+
+    if (!lemmaTranslations) {
+      // Use our own rules if lemmaTranslations are not provided
+      if (source === 'lookup') {
+        // For requests initiated by a lookup component:
+        if (textSelector.languageID === Constants.LANG_LATIN && this.app.state.lemmaTranslationLang) {
+          lemmaTranslations = { locale: this.app.state.lemmaTranslationLang }
+        }
+      } else {
+        // For requests initiated by text selection on a page
+        if (textSelector.languageID === Constants.LANG_LATIN &&
+          featureOptions.items.enableLemmaTranslations.currentValue &&
+          !featureOptions.items.locale.currentValue.match(/^en-/)) {
+          lemmaTranslations = { locale: featureOptions.items.locale.currentValue }
```

Comment:
This is essentially the same as the the code that updates app.state.lemmaTranslationLang (uicontroller updateLemmaTranslations).  I don't think there is any difference here between this and the code for the source=== lookup so it should just probably be the following without a test on the source

```
if (textSelector.languageID === Constants.LANG_LATIN && this.app.state.lemmaTranslationLang) {
          lemmaTranslations = { locale: this.app.state.lemmaTranslationLang }
}
```

shouldn't it?